### PR TITLE
Feature/mp-11

### LINKED ISSUE #17: MP-11 Email service
Добавить сервис отправки электронной почты
Добавить отправку письма с временным паролем

> Здравствуйте, Иванов Иван
> Ваш временный пароль: 1234

### PR DESCRIPTION
**Pull request by yanmayak**
Completed the task, added email service and tested on host smtp.yandex.ru. Added some test files to .gitignore, added dependency to pom.xml

### CODE REVIEW

**Review by ivamijava — APPROVED** [review]
K A I F

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
+src/main/resources/application-local.yaml
+ru/mambetpages/service/TestController.java
```
Comment (VaryaGet) [inline .gitignore]: Remove "ru/mambetpages/service/TestController.java" from gitignore, and just don't add to commit this file

**Review by VaryaGet — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (yanmayak) [reply .gitignore]: done

**Review by VaryaGet — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -3,6 +3,15 @@ spring:
     exclude:
       - org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
       - org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+  profiles:
+    active: local
+  mail:
+    host: smtp.yandex.ru
+    port: 465
+    username:
```
Comment (ArtemGet) [inline src/main/resources/application.yaml]: Better use arguments, like ${USERNAME} to all attributes

**Review by ArtemGet — CHANGES_REQUESTED** [review]

**Review by ArtemGet — APPROVED** [review]

**Pull request merged**